e2e/cypress/integration/messaging/post_textbox_height_spec.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Messaging', () => {
+    before(() => {
+        // # Login and navigate to town-square
+        cy.toMainChannelView('user-1');
+
+        // # Add two posts
+        getInputBox().type('test post 1{enter}');
+        getInputBox().type('test post 2{enter}');
```

Comment:
nit: We can use existing custom command `cy.postMessage` that almost doing the same thing.
```javascript
        cy.postMessage('test post 1');
        cy.postMessage('test post 2');
```

Reply:
Done :+1: